telemetry/definitions/commonDefinitions.json

```diff
@@ -3430,23 +3473,50 @@
             "passive": true
         },
         {
-            "name": "codeTransform_buildComplete",
-            "passive": true,
-            "description": "During the transformation progress, log the status of the build.",
+            "name": "codeTransform_dependenciesCopied",
+            "description": "Deprecated - use codeTransform_uploadProject (The repo copies over at least one dependency successfully)",
```

Comment:
could you also point to the new one along with deprecated

Reply:
codeTransform_uploadProject is the new metric (that will contain a field to indicate whether dependencies are copied)